src/modules/link-records/link-records.component.ts

```diff
@@ -60,6 +60,7 @@ export class SkyLinkRecordsComponent implements OnInit, AfterContentInit, OnDest
   @Input() public itemTitleTemplate: TemplateRef<any>;
   @Input() public keyIdSelector: string = 'id';
   @Input() public selectedByDefault: boolean = true;
+  @Input() public showNewFieldValues: boolean;
```

Comment:
this should be initialized to true (and so remove the boolean type, itll be inferred when you assign)